### DISCUSSION TITLE
BlazorBindGen and Game Engine

### DISCUSSION THREAD

**Discussion by GeorgeS2019** [sps014/BlazorBindGen#10] [Ideas]
I wonder if there is interest to adapt and explore how to use BlazorBindGen with a Game exported as WASM?

Currently, there is such an initiative using Godot exported as WASM within a Blazor framework. It would be interesting to explore how BlazorBindGen could contribute to such an [initiative](https://github.com/caemanswartz/SharpScapeGodotGame/issues/3#issue-1286888474)?

**Comment by GeorgeS2019** [comment]
Here is the shared architecture for anyone interested to explore this new possibility of interfacing a cross-platform Game Engine (in addition to BabylonJS) using Blazor.
![sharpscape-architecture](https://user-images.githubusercontent.com/49812372/176117032-0573c8dc-cbc5-4387-8e1f-72adfdcd9171.svg)

**Comment by sps014** [comment]
WebGL is already in roadmap of [SILK.NET](https://github.com/dotnet/Silk.NET), It will be game changer for Blazor.

**Comment by GeorgeS2019** [comment]
I know at least one GAME which was fully developed using SILK.NET has now switched to Godot :-)
